README.md

```diff
@@ -77,8 +70,8 @@ const orderSchema = require('./order-schema');
 
 // merge conflicting "Query", "Mutation", and "Subscription" definitions
 const typeDefs = mergeTypeDefs([
-  ...customerSchema.types,
-  ...orderSchema.types
+  customerSchema.typeDefs,
+  ...orderSchema.typeDefs
```

Comment:
We don't need the spread operator here, right? The expectation is that you'll define all typeDefs in a single AST rather than an array of them?